test/lib/ansible_test/_internal/util_common.py

```diff
@@ -269,7 +269,10 @@ def named_temporary_file(args: CommonConfig, prefix: str, suffix: str, directory
             tempfile_fd.write(to_bytes(content))
             tempfile_fd.flush()
 
-            yield tempfile_fd.name
+            try:
+                yield tempfile_fd.name
+            finally:
+                pass
```

Comment:
What's the purpose of `pass` here?

Reply:
It's a work-around for https://pylint.pycqa.org/en/latest/user_guide/messages/warning/contextmanager-generator-missing-cleanup.html -- I'm not pleased with this approach, but I wanted to unblock other work rather than spend more time on the issue.